scipy/integrate/tests/test_quadrature.py

```diff
@@ -358,19 +358,66 @@ def test_cumtrapz(self):
 
 
 class TestTrapezoid:
-    """This function is tested in NumPy more extensive, just do some
-    basic due diligence here."""
-    def test_trapezoid(self):
-        y = np.arange(17)
-        assert_equal(trapezoid(y), 128)
-        assert_equal(trapezoid(y, dx=0.5), 64)
-        assert_equal(trapezoid(y, x=np.linspace(0, 4, 17)), 32)
-
-        y = np.arange(4)
-        x = 2**y
-        assert_equal(trapezoid(y, x=x, dx=0.1), 13.5)
-
-    def test_trapz(self):
+    def test_simple(self):
```

Comment:
is this the entirety of the numpy tests?

Reply:
Yes, these are all `trapz` tests. They are taken from here: https://github.com/numpy/numpy/blob/e8e523f9d193e7d1136bb94e7b316c74922c5a97/numpy/lib/tests/test_function_base.py#L2118